build/mac/DS_Store-files/main.py

```diff
@@ -0,0 +1,84 @@
+from tempfile import TemporaryDirectory
+from uuid import uuid4
+
+
+def main():
+    for channel in ('', 'Nightly', 'Beta', 'Release'):
+        app_name = 'Brave Browser' + (f' {channel}' if channel else '')
+        ds_store = 'DS_Store' + (f'.{channel.lower()}' if channel else '')
+        create_ds_store(app_name, '../dmg-background.png', f'{ds_store}')
+
+
+def create_ds_store(app_name, bg_file, ds_store_path):
+    with TemporaryDirectory() as temp_dir:
+        # We create and mount a temporary DMG because this creates an HFS+ file
+        # system, which is required for at least Alias.for_file(...) below. See
+        # github.com/dmgbuild/mac_alias/issues/22#issuecomment-1350790003.
+        dmg_path = join(temp_dir, f'{app_name}.dmg')
+        with TemporaryDirectory() as empty:
+            check_call([
+                'hdiutil', 'create', '-fs', 'HFS+', '-format', 'UDRW',
+                '-sectors',
+                str(8 * 1024 * 1024), '-volname', app_name, '-srcfolder',
+                empty, '-quiet', dmg_path
+            ])
+        mount_path = join('/Volumes', basename(__file__) + str(uuid4()))
+        check_call([
+            'hdiutil', 'attach', '-mountpoint', mount_path, '-noautoopen',
+            '-quiet', dmg_path
+        ])
+        try:
+            symlink('/Applications', join(mount_path, 'Applications'))
+            background_dir = join(mount_path, '.background')
+            makedirs(background_dir)
+            copyfile(bg_file, join(background_dir, 'background.png'))
+            ds_store_tmp = join(mount_path, '.DS_Store')
+            with DSStore.open(ds_store_tmp, 'w+') as d:
+                d[app_name + '.app']['Iloc'] = (150, 155)
+                d[' ']['Iloc'] = (430, 150)
+                d['.']['bwsp'] = {
+                    'ContainerShowSidebar': False,
+                    'ShowPathbar': False,
+                    'ShowSidebar': False,
+                    'ShowStatusBar': False,
+                    'ShowTabView': False,
+                    'ShowToolbar': False,
+                    'WindowBounds': '{{548, 815}, {602, 350}}'
+                }
+                bg_alias = Alias.for_file(
+                    join(background_dir, 'background.png'))
+                d['.']['icvp'] = {
+                    'arrangeBy': 'none',
+                    'backgroundColorBlue': 1.0,
+                    'backgroundColorGreen': 1.0,
+                    'backgroundColorRed': 1.0,
+                    'backgroundType': 2,
+                    'backgroundImageAlias': bg_alias.to_bytes(),
+                    'gridOffsetX': 0.0,
+                    'gridOffsetY': 0.0,
+                    'gridSpacing': 100.0,
+                    'iconSize': 128.0,
+                    'labelOnBottom': True,
+                    'showIconPreview': True,
+                    'showItemInfo': False,
+                    'textSize': 16.0,
+                    'viewOptionsVersion': 1
+                }
```

Comment:
I've no idea what's going on here. Is it something that we'd like to do to make our DMG look a bit prettier, but never did?

Reply:
Does the following from this PR description, together with `build/mac/DS_Store-files/README.md` help?

> This PR adds new Python code in DS_Store-files/. The DS_Store files configure the visual appearance of Brave's DMG installer, such as the background image and icon locations. It was necessary to update those DS_Store files to reflect a slightly different file structure produced by upstream's DMG code. DS_Store is a proprietary format and it was hard to edit the existing files. The Python code in DS_Store-files/ solves this problem: It can be used to generate the DS_Store files from scratch, with explicit and deterministic settings.

Reply:
1. I renamed `main.py` to `update_ds_store_files.py` in ed86cdac5a0bfc8675eac84a6db5c6479acba6f0.
2. I added a header comment that `update_ds_store_files.py` is not used by the build in c3bb1364615f5d566500aee623d1494a03c0aabf.
3. I found the `DS_Store` parameters scattered in various places online, then modified them through trial and error to obtain the same appearance as our DMGs before this PR. I added a comment in 88e4ef822fcae4d1dbe48824c69e01b5a5903004 that says that the parameters were gleaned online, but cannot really offer much more insight into them.